scripts/build/generateDocs.mjs

```diff
@@ -245,10 +245,12 @@ export async function processDocFiles(remoteReadmeVersion = 'master', readmeVari
   // process
   // README.md
 
+  Logger.warn('WARNING: overwrite is set to FALSE for README.md - please update this when template changes are merged');
```

Comment:
Since the template is remote, individual repos wouldn't know if the template is newer than their local copy. And anyone who happens to be running the doc build in a component probably wouldn't know either.

Why not copy the template each time we build and remove any chance it's missed?

Reply:
Hmmmm are you suggesting if overwrite: false is passed in, we just pull the latest file, throw it into docTemplates to live there and then don't copy it into place? That would work 🤔

We modify the destination file now so this wouldn't change a lot in the code, just change what's lumped into which boolean condition

Reply:
I'm asking if "overwrite" should default to true for the readme.

Reply:
Closing since we resolved this offline! Basically we need to wait until the remote README template has been updated to include at least the path changes required for the new scripting